### PR TITLE
[feat](thrift) add NereidsId and cardinality to TPlanNode

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
coauthor: [Mryange](https://github.com/Mryange)

1. set nereids_id in TPlanNode (thirft)
2. print numNulls in column statistics
3. print table stats in profile(before physical plan)

let BE print these info into profile

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java [hunk 1]
```diff
@@ -291,8 +291,8 @@ public boolean enclosed(ColumnStatistic other) {
     @Override
     public String toString() {
         return isUnKnown ? "unknown(" + count + ")"
-                : String.format("ndv=%.4f, min=%f(%s), max=%f(%s), count=%.4f, avgSizeByte=%f",
-                ndv, minValue, minExpr, maxValue, maxExpr, count, avgSizeByte);
+                : String.format("ndv=%.4f, min=%f(%s), max=%f(%s), count=%.4f, numNulls=%d, avgSizeByte=%f",
+                ndv, minValue, minExpr, maxValue, maxExpr, count,numNulls, avgSizeByte);
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java:295]:
```suggestion
                ndv, minValue, minExpr, maxValue, maxExpr, count, numNulls, avgSizeByte);
```

**Comment by englefly** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java (hunk 1 above)
Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java]: done

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run performance

**Review by Mryange — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**